onnxruntime/core/providers/nuphar/scripts/symbolic_shape_infer.py

```diff
@@ -240,7 +274,34 @@ def _onnx_infer_single_node(self, node):
             vi.CopyFrom(self.tmp_mp_.graph.output[i_o])
             self.known_vi_[vi.name] = vi
 
-    def _get_int_values(self, node):
+    def _onnx_infer_subgraph(self, node, subgraph):
+        if self.verbose_ > 2:
+            print('Inferencing subgraph of node {} with output({}...): {}'.format(node.name, node.output[0], node.op_type))
+        assert len(node.input) == len(subgraph.input)
+        subgraph_implicit_input = set()
```

Comment:
Maybe a comment calling out that the node inputs are not passed directly to the subgraph would avoid potential confusion about the slightly indirect relationship between these lengths.